python/ccxt/pro/whitebit.py

```diff
@@ -535,7 +535,7 @@ def parse_ws_order(self, order, market=None):
         marketId = self.safe_string(order, 'market')
         market = self.safe_market(marketId, market)
         id = self.safe_string(order, 'id')
-        clientOrderId = self.omit_zero(self.safe_string(order, 'client_order_id'))
```

Comment:
Please update ts file because the code was transpiled automatically and this update would be rewritten.